modules/creating-logfilesmetricexporter.adoc

```diff
@@ -0,0 +1,80 @@
+// Module included in the following assemblies:
+//
+// * logging/log_collection_forwarding/cluster-logging-collector.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="creating-logfilesmetricexporter_{context}"]
+= Creating a LogFileMetricExporter resource
+
+In {logging} version 5.8 and newer versions, the `LogFileMetricExporter` custom resource (CR) is no longer deployed with the collector by default.
```

Comment:
```
In logging version 5.8 and newer versions, the LogFileMetricExporter is no longer deployed with the collector by
 default.  You must manually create a `LogFileMetricExporter` custom resource (CR) to generate metrics from the
 logs produced by running containers.
```

The subtly is it was a container added to the collector deployment and no CR existed prior to 5.8.  Not its deployment is solely controlled by the CR